dev/source/docs/code-overview-scheduling-your-new-code-to-run-intermittently.rst

```diff
@@ -12,11 +12,11 @@ Running your code with the scheduler
 
 The most flexible way to run your code at a given interval is to use the
 scheduler.  This can be done by adding your new function to the
-`scheduler_tasks <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/ArduCopter.cpp#L96>`__
+`scheduler_tasks <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L91>`__
 array in
-`ArduCopter.cpp <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/ArduCopter.cpp>`__. 
-Note that there are actually two task lists, `the upper list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/ArduCopter.cpp#L788>`__
-is for high speed CPUs (i.e. Pixhawk) and the `lower list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/ArduCopter.cpp#L856>`__
+`Copter.cpp <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp>`__. 
+Note that there are actually two task lists, `the upper list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L788>`__
+is for high speed CPUs (i.e. Pixhawk) and the `lower list <https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/Copter.cpp#L856>`__
```

Comment:
I couldn't find 2 different task lists, therefore left it as is
But needs to be fixed